website/content/labs/lab1.md

```diff
@@ -30,30 +43,34 @@ Log into https://mit.cs.uchicago.edu/. Under "Your projects", you should see an
   cd cs220
   ```
 
-* Inside that folder, create a file called `test.txt` with your full name.
+* Inside that folder, create a file called `README.md` with your full name.
 
-* On the GitLab project page (the one that says "The repository for this project is empty") there is pull-down list with "SSH" selected. Make sure to change this to "HTTPS".
+* On your repository's GitHub page, right under "Quick setup — if you’ve done this kind of thing before" there is a URL field with two buttons: HTTPS and SSH. Make sure to change this to "HTTPS".
 
-Now, from inside your CMSC 22000 directory, run the instructions for "Git global setup" and for "Existing folder". If any commands asks you to enter a username and password, just enter your CNetID and password. Don't worry about what each individual command does; we will be seeing what most of these commands do in this lab. However, if you encounter any error messages, please make sure to let us know.
+Now, from inside your CMSC 22000 directory, run the commands that appear under "...or create a new repository on the command line" *except* the first one (the one that starts with `echo`. 
 
-You can verify that your repository was correctly set up by going back to https://mit.cs.uchicago.edu/. If you go to your repository page, you should now see it contains a `test.txt` file. If you click on it, you can see its contents.
+*Note: If any commands asks you to enter a username and password, just enter your GitHub username and password* (do not enter your CNetID and password)
+ 
+Don't worry about what each individual command does; we will be seeing what most of these commands do in this lab. However, if you encounter any error messages, please make sure to let us know.
+
+You can verify that your repository was correctly set up by going back to your repository's page on GitHub, you should now see it contains a `README.md` file. If you click on it, you can see its contents.
 
 ## Creating a commit
 
 If you make changes to your repository, the way to store those changes (and the updated versions of the modified files) is by creating a *commit*. So, let's start by making some changes:
 
-* Edit `test.txt` to also include your CNetID
-* Add a new file called `README.md`. This file should contain a single line with the text `Hello, world!`
+* Edit `README.md` to also include your CNetID
+* Add a new file called `test.txt`. This file should contain a single line with the text `Hello, world!`
 
 Creating a commit is a two-step process. First, you have to indicate what files you want to include in your commit. Let’s say we want to create a commit that only includes the updated `test.txt` file. We can specify this operation explicitly using the `git add` command from the terminal:
```

Comment:
Here the write-up talk about only including the `test.txt` file, but then continues to add `README.md`. As far as I understand, this sentence should say `README.md` instead of `test.txt` (or vice versa; the continuing section of the write-up should be changed to add and commit `test.txt`)

Reply:
Pushed a fix for this.